src/test/java/org/mockitousage/junitrule/StubbingWarningsJUnitRuleTest.java

```diff
@@ -0,0 +1,257 @@
+package org.mockitousage.junitrule;
```

Comment:
Why is this test class in the `mockitousage` package? Isn't it a unit test for JUnitRule?